packages/dx-react-grid/docs/reference/column-chooser.md

```diff
@@ -29,22 +29,22 @@ A value with the following shape:
 
 Field | Type | Description
 ------|------|------------
-visible | boolean | Specifies whether an overlay is visible.
-target | ReactComponent | The React component that is used to set the position of the overlay.
+visible | boolean | Specifies whether the overlay is visible.
+target | ReactComponent | A React component that is used for overlay positioning.
 onHide | () => void | An event that initiates overlay hiding.
 children | Array&lt;ReactElement&gt; | React elements used to render overlay content.
 
 ### ColumnChooserToggleButtonProps
 
-Describes properties passed to a component that renders the column chooser toggle button.
+Describes properties passed to a component that renders a button that invokes the column chooser.
 
 A value with the following shape:
 
 Field | Type | Description
 ------|------|------------
 onToggle | () => void | An event that initiates overlay showing or hiding.
-getMessage | ([messageKey](#localization-messages): string) => string | Returns title for the column chooser toggle button. Available in the "@devexpress/dx-react-grid-material-ui" package.
-buttonRef | (ref: ReactElement) => void | A function that take a React element of the root element.
+getMessage | ([messageKey](#localization-messages): string) => string | Returns the toggle button's text. Available in the "@devexpress/dx-react-grid-material-ui" package.
```

Comment:
Why do we write that the getMessage function that receives the message key argument returns only the single message? It feels strange.

Reply:
Looks like it should sound like "Returns a specified localization message."